### PR TITLE
Update Tourian Escape Room 4

### PR DESCRIPTION
**Pull request by kjbranch**
Eventually we can add some sort of tech or strats to killing and dodging the tedious space pirates in rooms 3 and 4.

### CODE REVIEW

File: region/tourian/main.json [hunk 1]
```diff
@@ -3876,13 +3879,43 @@
...
+                      "HiJump"
+                    ]},
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      {"and": [
+                        "SpaceJump",
+                        {"or": [
+                          "HiJump",
+                          {"and": [
+                            "canCarefulJump",
```
Comment (kjbranch) [inline region/tourian/main.json:3903]: is this doable without acid frames with space jump and no wall jumps?

File: region/tourian/main.json [hunk 2]
```diff
@@ -3876,13 +3879,43 @@
...
+                      "canCarefulJump",
+                      "ScrewAttack",
+                      "Morph",
+                      {"enemyDamage": {
+                        "enemy": "Tourian Space Pirate (all)",
+                        "type": "contact",
+                        "hits": 1
+                      }}
+                    ]},
+                    {"or": [
```
Comment (osse101) [inline region/tourian/main.json:3891]: What part of the room is this `or` helping with?

File: region/tourian/main.json (hunk 1 above)
Comment (osse101) [reply region/tourian/main.json:3903]: With min maxing you could.  It's probably better to have the acid frames here.  Until there is a `canOptimizeMovement` tech.

File: region/tourian/main.json (hunk 2 above)
Comment (kjbranch) [reply region/tourian/main.json:3891]: that's for climbing the central shaft while avoiding pirate lasers

Comment (osse101) [reply region/tourian/main.json:3891]: If you kill the right side pirates then stick to the right, I don't know that there are lazers to dodge.  But I like keeping these options here.

File: region/tourian/main.json [hunk 3]
```diff
@@ -3876,13 +3879,43 @@
...
+                          ]},
+                          {"acidFrames": 100}
+                        ]}
+                      ]}
                     ]}
                   ]
                 }
               ],
-              "note": "The acid starts rising when Samus is at the bottom of the long climb"
+              "note": "The acid starts rising when Samus is at the bottom of the long climb."
```
Comment (osse101) [inline region/tourian/main.json:3918]: A practical note on baiting the right side pirates to shoot right before you go up main shaft would be nice, but logically you can just kill them 🤔.
And a strat for falling through left pirates depends somewhat on how fast and on which side you climb main shaft.

**Review by osse101 — APPROVED** [review]

**Pull request merged**